packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -195,6 +195,82 @@ enum AnimationBehavior {
 /// ```
 /// {@end-tool}
 ///
+/// {@tool dartpad --template=freeform}
+///
+/// ```dart imports
+/// import 'package:flutter/material.dart';
+/// ```
+///
+/// ```dart
+/// void main() {
+///   runApp(AnimationControllerDemo());
+/// }
+///
+/// class AnimationControllerDemo extends StatefulWidget {
+///   @override
+///   _AnimationControllerDemoState createState() =>
+///       _AnimationControllerDemoState();
+/// }
+///
+/// class _AnimationControllerDemoState extends State<AnimationControllerDemo>
+///     with SingleTickerProviderStateMixin {
+///   static const Duration _duration = Duration(seconds: 1);
+///   dynamic controller;
```

Comment:
Fields should be properly typed.

Reply:
I made it dynamic to fix null safety , As analyse linux test was failing.